iaso/models/org_unit.py

```diff
@@ -240,6 +240,14 @@ def filter_for_user_and_app_id(
 OrgUnitManager = models.Manager.from_queryset(OrgUnitQuerySet)
 
 
+def get_creator_name(creator):
+    if creator is None:
+        return None
+    if creator.first_name or creator.last_name:
+        return f"{creator.username} ({creator.first_name} {creator.last_name})"
```

Comment:
```suggestion
        return f"{creator.username} ({creator.get_full_name()})"
```

Just in case, there is this method.